aws/sqs.go

```diff
@@ -0,0 +1,79 @@
+package aws
+
+import (
+	"strconv"
+	"time"
+
+	awsgo "github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/sqs"
+	"github.com/gruntwork-io/cloud-nuke/logging"
+	"github.com/gruntwork-io/gruntwork-cli/errors"
+)
+
+// Returns a formatted string of SQS Queue URLs
+func getAllSqsQueue(session *session.Session, region string, excludeAfter time.Time) ([]*string, error) {
+	svc := sqs.New(session)
+	result, err := svc.ListQueues(&sqs.ListQueuesInput{})
+	if err != nil {
+		return nil, errors.WithStackTrace(err)
+	}
+
+	var urls []*string
+
+	for _, queue := range result.QueueUrls {
+		param := &sqs.GetQueueAttributesInput{
+			QueueUrl:       queue,
+			AttributeNames: awsgo.StringSlice([]string{"All"}),
```

Comment:
NIT: Do we need all, or is there a way to request solely `CreatedTimestamp`, as that seems to be all we're using?

Reply:
Great! done 😅